front/src/views/About.vue

```diff
@@ -1,16 +1,24 @@
 <template>
-  <div class="about">
-    #about
-  </div>
+    <div class="about">
+        <ul>
+            <li>version: {{ mode }}.{{ version }}</li>
```

Comment:
Maybe we can id too to the footer ?

Reply:
Fur sure in a next version ?

![](https://media2.giphy.com/media/DmRu4qCv39160/giphy.gif?cid=5a38a5a240a13e95c9d72a90f26b8634c6d3c20edcbc39d0&rid=giphy.gif)